inception-recommendation/src/main/java/de/tudarmstadt/ukp/inception/recommendation/sidebar/RecommendationSidebar.html

```diff
@@ -49,6 +49,8 @@ <h3 class="panel-title">Recommendation Information</h3>
         <input wicket:id="save" type="submit" class="btn btn-primary"  wicket:message="value:apply"/>
       </div>
     </form>
+	<div wicket:id="chart-container" id="chart-container-js"></div>
```

Comment:
We must not set the HTML `id` attribute ourselves. Wicket has to do it and we can request this by calling `setOutputMarkupId(true)` on the associated Wicket component. That way we can ensure that we can put as many instances of a component on a page as we wish.

Reply:
I put the chart div outside the flex-content, and it still doesn't show.

Reply:
Try giving it an explicit minimum height.

Reply:
Not working. I tried a few more things to the div and no luck. Finally, while fiddling with the css of the page I noticed when the script 
`<link rel="stylesheet" type="text/css" href="./wicket/resource/de.tudarmstadt.ukp.clarin.webanno.brat.resource.BratCssVisReference/style-vis-ver-1D8B90DD5044623823E65D2592A1297C.css">`

is removed from the head of the page then the graph shows up. Does this give any clue?

Reply:
There may be a clash between the CSS defined in that style sheet and the CSS required by the graph - but it needs closer investigation. The style-vis.css might be too broad and take over all SVG elements on a page including the graph.